write_test.go

```diff
@@ -40,11 +40,11 @@ func TestRoaringSizes(t *testing.T) {
 		{[]uint32{0, 1}, 20, 20},
 		{[]uint32{0, 10000000}, 28, 28},
 
-		{[]uint32{0, 1, 2}, 22, 15},
+		{[]uint32{0, 1, 2}, 22, 22},
```

Comment:
just curious, why do we have update the unit test here?

Reply:
It seems the roaring bitmaps's `RunOptimize` has changed a little over the versions.